src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -114,6 +114,13 @@ export function formatPercentage(
     );
 }
 
+export function getProfiledCount(
```

Comment:
not a big deal but you probably don't need to export this since it's used internally only.  it's sort of equivalent to private.   otherwise it pollutes the global namespace.  but no need to fix.